frontend/src/features/qa/page-list/page-list.ts

```diff
@@ -28,9 +32,13 @@ const sortableFields = {
     defaultDirection: "desc",
   },
   approved: {
-    label: msg("Review Status"),
+    label: msg("Approval"),
```

Comment:
Not sure if this is the most accurate title for the column?  "Page status?"  I liked "Review Status"...

Reply:
We discussed "Review" referring to reviewing the whole crawl, since the process of approving/rejecting is called an approval process.

Reply:
Looks like I've been bitten by my own problem of not keeping the mock-ups up to date this round!  Keep as is if I've just forgotten. 👍